src/js/Setup.js

```diff
@@ -256,8 +256,8 @@ function createCindyNow() {
 }
 
 function loadExtraModules() {
```

Comment:
Just deleted the line which were commented: https://github.com/CindyJS/CindyJS/pull/304/commits/ad3e024fd3f2bb4a37de79ac3a92032adefd94da